appmgr/src/install/mod.rs

```diff
@@ -997,6 +996,8 @@ pub async fn install_s9pk<R: AsyncRead + AsyncSeek + Unpin>(
             *main_status = prev.status.main;
             main_status.save(&mut tx).await?;
         }
+        remove_current_dependents(&mut tx, pkg_id, prev.current_dependencies.keys()).await?; // remove previous
+        add_current_dependents(&mut tx, pkg_id, &current_dependencies).await?; // add new
```

Comment:
maybe `remove_from_previous_dependencies_dependents`, `add_to_dependencies_current_dependents`

Reply:
or `remove from previous set of dependencies`, `add to new set of dependencies`